ViteMaDose/Models/FollowedCentre.swift

```diff
@@ -16,6 +16,5 @@ public extension FollowedCentre {
     enum NotificationsType: String, Codable, Hashable {
         case none
         case all
-        case chronodoses
```

Comment:
`chronodoses` is part of a Codable enum. Are we sure it's now never ever returned by API? (To avoid a crash when decoding an unknown value)